libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -552,6 +568,46 @@ uint16_t AP_MotorsHeli_Single::calculate_ddfp_output(float yaw_out)
     return ret;
 }
 
+// if using DDFP tail we need to get the pwm range and trim of the servo channel for use later
+bool AP_MotorsHeli_Single::update_ddfp_range_and_trim(void)
+{
+    if (_tail_type != AP_MOTORS_HELI_SINGLE_TAILTYPE_DIRECTDRIVE_FIXEDPITCH_CW && _tail_type != AP_MOTORS_HELI_SINGLE_TAILTYPE_DIRECTDRIVE_FIXEDPITCH_CCW) {
+        return true;
+    }
+
+    // find the channel that motor 4 is assigned to
+    uint8_t chan;
+    if (!SRV_Channels::find_channel(SRV_Channel::k_motor4, chan)) {
+        // We have failed to find the tail motor channel
+        gcs().send_text(MAV_SEVERITY_CRITICAL, "DDFP Error: Motor 4 not assigned");
```

Comment:
Can't have send texts's in here, its running at loop rate. Or at least not with out some logic to not spam.